prep-html-links/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Home Page</title>
+  </head>
+  <body>
+    <h1>Home page</h1>
+    <a
+      href="file:///C:/Users/spazz/repos/prep-1121-code-solutions/prep-html-links/contact-us/contact-us.html"
+      >Click here to go to the contact page</a
+    >
```

Comment:
Since your `index.html` file is in the same directory as your `contact-us/` directory, you can use a relative path to `contact-us/contact-us.html`. That way anybody else can run your code without having the exact same file system structure as your computer. For example:

```html
<a href="contact-us/contact-us.html">Click here to go to the contact page</a>
```